### ISSUE TITLE
FR: Updating the Linking word

### ISSUE THREAD

**Issue by FahimFBA** [firebase/firebase-ios-sdk#9019]
### Feature proposal

* Updating the linking words in the `README.md` file in the `master` branch.

I want to update the Linking words in the README file. I want to remove the `.md` extension from the linking words as that is going to be linked in the URL box anyway. Please follow the code below:

```diff
+ See [HeadersImports](HeadersImports.md) Markdown file.
- See [HeadersImports.md](HeadersImports.md).
```

I have already forked and made the changes. If you approve the update then I would create a PR.

**Comment by FahimFBA** [comment]
I have created a PR for this Issue.

**Comment by paulb777** [comment]
Fixed in #9020
